lib/core/README.md

```diff
@@ -0,0 +1,13 @@
+# Storybook for React
+
+[![Greenkeeper badge](https://badges.greenkeeper.io/storybooks/storybook.svg)](https://greenkeeper.io/)
```

Comment:
outdated readme

Reply:
Fixed, though somehow it's not showing up here. Still need to write the actual contents and it's in the TODO list at top of PR.